db/migrate/20240418224031_add_image_to_medical_records.rb

```diff
@@ -0,0 +1,4 @@
+class AddImageToMedicalRecords < ActiveRecord::Migration[7.1]
+  def change
```

Comment:
Why have you created this migration?

Reply:
@davepcage I think a better english is "Why did you create this migration?"

Reply:
You don't need a new migration for "image", because you are using ActiveRecordStorage table, so you can delete this file.

Reply:
Thanks!